_data/people/savvy379.yml

```
@@ -5,4 +5,16 @@ active: green
 institution: Princeton University
 website: 
 photo: /assets/images/team/Savannah-Thais.jpg
-presentations:
\ No newline at end of file
+presentations:
+  - title: "Introduction and Plans"
+    date: 09/12/19
```

```suggestion
    date: 2019-09-12
```

ISO 8061, please. :)